CHANGELOG.md

```diff
@@ -0,0 +1,26 @@
+# Collaborative Changelog
```

Comment:
Nice 👏🏼 

Helpful to see all the changes (feel free to put this in a PR instead next time if that's easier, then we don't have to keep this file checked-in to the codebase)

Reply:
I'm going to copy this into the PR description